troubleshooting/troubleshooting.md

```diff
@@ -810,7 +810,7 @@ You can choose different values to enable or disable certain flags of SIP. Some
 * `03000000` - Disable kext signing (0x1) and filesystem protections (0x2).
 * `FF030000` - Disable all [flags in macOS High Sierra](https://opensource.apple.com/source/xnu/xnu-4570.71.2/bsd/sys/csr.h.auto.html) (0x3ff).
 * `FF070000` - Disable all [flags in macOS Mojave](https://opensource.apple.com/source/xnu/xnu-4903.270.47/bsd/sys/csr.h.auto.html) and in [macOS Catalina](https://opensource.apple.com/source/xnu/xnu-6153.81.5/bsd/sys/csr.h.auto.html) (0x7ff) as Apple introduced a value for executable policy.
-* `FF0F0000` - Disable all flags in macOS Big Sur (0xfff) which has another new [flag for authenticated root](https://eclecticlight.co/2020/06/25/big-surs-signed-system-volume-added-security-protection/).
+* `77080000 ` - Disable all flags in macOS Big Sur (0xfff) which has another new [flag for authenticated root](https://eclecticlight.co/2020/06/25/big-surs-signed-system-volume-added-security-protection/).
```

Comment:
How does SIP's total bits become smaller from Catalina? No bits were dropped in Big Sur

Reply:
I struggle with the same question. But when I check:
```
csrutil authenticated-root
Authenticated Root status: disabled
csrutil status            
System Integrity Protection status: disabled.
```
See here https://forums.macrumors.com/threads/opencore-on-the-mac-pro.2207814/post-28714864

Reply:
csr-active-config=FF0F0000
```
nvram 7C436110-AB2A-4BBB-A880-FE41995C9F82:csr-active-config
nvram: Error getting variable - '7C436110-AB2A-4BBB-A880-FE41995C9F82:csr-active-config': (iokit/common) data was not found
ioreg -p IODeviceTree -l | grep csr-active-config (empty)
csrutil status
System Integrity Protection status: unknown (Custom Configuration).

Configuration:
	Apple Internal: disabled
	Kext Signing: disabled
	Filesystem Protections: disabled
	Debugging Restrictions: disabled
	DTrace Restrictions: disabled
	NVRAM Protections: disabled
	BaseSystem Verification: disabled

This is an unsupported configuration, likely to break in the future and leave your machine in an unknown state.
csrutil authenticated-root
Authenticated Root status: disabled
```